src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/CLHSDB.java

```diff
@@ -217,8 +225,9 @@ private void attachDebugger(final String executablePath, final String corePath)
     private void connect(final String remoteMachineName) {
         // Try to open this core file
         try {
-            System.err.println("Connecting to debug server, please wait...");
+            System.out.println("Connecting to debug server, please wait...");
             agent.attach(remoteMachineName);
+            this.remoteMachineName = remoteMachineName;
```

Comment:
I'm a little surprised to see that there is already some support for connecting to a debug server in CLHSDB.java. Was it previously used from anywhere?

Reply:
I cannot find out caller of `CLHSDB::connect`. I guess CLHSDB.java is copied from HSDB.java because they are similar - HSDB uses `connect()` to connect to debug server.